scripts/cover.sh

```diff
@@ -41,6 +46,9 @@ done
 i=0
 start_waitpids
 for pkg in "$@"; do
+  if ! ls "${GOPATH}/src/${pkg}" | grep _test\.go$ >/dev/null; then
+    continue
+  fi
```

Comment:
Indentation is inconsistent with the rest of the file (plus in the waitpids change).